manifest/net.dfgHiatus/VRMLImporter/info.json

```diff
@@ -0,0 +1,43 @@
+{
+	"name": "VRMLImporter",
+	"id": "net.dfgHiatus.VRMLImporter",
+	"description": "Allows importing VRML 1.0/2.0 and X3D files. Requires Blender to be installed on the system or in [RESONITE PATH]/tools/Blender.",
+	"category": "Asset Importing",
+	"sourceLocation": "https://github.com/dfgHiatus/ResoniteVRMLImporter",
+	"versions": {
```

Comment:
Add "platforms" field to specify this mod is windows only as it relies on exe files

Reply:
This should work on Linux machines as well, provided they have blender installed on their system path

Reply:
I guess it just wouldn't be 'fully' working in that case because of `vrml1tovrml2.exe` and `vr1tovr2.exe` (although this one seems to be unused)

Reply:
Ah yes, if one were to build the Linux binaries, then it would be truly cross-platform. Mb

Reply:
Not sure if wine would pick that up or work for it. If it does work on linux or at least should then specifying platforms won't be needed.

Reply:
Wine would probably work for it, but Blender would almost certainly need to be in the Resonite tools directory instead of a system install. Maybe it should be `platforms: ['windows', 'linux-wine']`?

Reply:
Could also just add the caveat that VRML 1.0 support only works on Windows...

Reply:
I'd say specifying the platforms that it should be fully supported by should be listed. Not sure if it works at all on linux native or if it is only partially working there

Reply:
After some thought, let's put it as `platforms: ['windows', 'linux-wine']`. I think that's fitting